src/status_im/contexts/wallet/create_account/new_keypair/backup_recovery_phrase/view.cljs

```diff
@@ -35,15 +35,15 @@
    [quo/text {:style {:margin-left 12}} (i18n/label item)]])
 
 (defn- view-internal
-  [{:keys [theme]}]
+  [{:keys [revealed? theme]}]
```

Comment:
perhaps `default-revealed?` is better, not particularly a fan of reassigning vars within the body then.